### DISCUSSION TITLE
l

### DISCUSSION THREAD

**Discussion by MohamedAmirTriguiAmirTrigui** [jertel/elastalert2#1161] [Q&A] [answered]
Hi all

I want to integrate Thehive with ELK using elastalert2 to send alerts!
So i need to use TheHive4py (is a Python API client for TheHive)
 I use this rule to do this this integration https://elastalert2.readthedocs.io/en/latest/ruletypes.html#thehive

but didn't worked
but when i use the thehive alert of elastalert it's worked but with no observable or description 
any help or better rules!

**Comment by jertel** [comment, marked as answer]
Please read #11 and provide sufficient information so someone can assist you.
